examples/sri-1/components/workgroup/workgroup/workgroupComponent.interface.ts

```diff
@@ -0,0 +1,24 @@
+import { Workstep } from '../workstep/workstep';
+import { Workgroup } from './workgroup';
+import { BpiSubject } from '../identity/bpiSubject';
+import { Agreement } from '../storage/agreement';
+import { Invitation } from '../invitation/invitation';
+import { Security } from '../policy/security';
+import { Privacy } from '../policy/privacy';
+
+export interface IWorkgroupComponent {
+
+    workgroups: Workgroup[];
+    invitations: Invitation[];
+    archivedWorkgroups: Workgroup[];
+    getWorkgroups(): Workgroup[];
+    createWorkgroup(name: string, id: string, owner: BpiSubject, worksteps: Workstep[], securityPolicy: Security, privacyPolicy: Privacy): Workgroup;
```

Comment:
It would be better to pass in a workgroup Type as param

Reply:
Done